executor/poolmgr/gp.go

```diff
@@ -519,6 +519,33 @@ func (gp *GenericPool) createPool() error {
 								"-secret-dir", gp.sharedSecretPath,
 								"-cfgmap-dir", gp.sharedCfgMapPath,
 								gp.sharedMountPath},
+							ReadinessProbe: &apiv1.Probe{
```

Comment:
I thought you told me that this can't be added, because existing envs would break?

Reply:
I added it only to fetcher container. We must soon make it mandatory for all environments too. right?